views/welcome.ejs

```diff
@@ -1,24 +1,57 @@
 <!DOCTYPE html>
 <%- include('./partials/head') %>
 <html lang="en">
+
+<style>
+  .db-card {
+    border: solid 1px rgb(0,0,0,0);
+    margin-top: 15px;
+    margin-bottom: 15px;
+    padding: 10px;
+    padding-bottom: 20px;
+    border-radius: 5pX;
+    cursor: pointer;
+  }
+
+  .db-card:hover {
+    border: solid 1px rgb(10, 70, 120);
+  }
```

Comment:
These css makes it look like the cards are clickable but no script is listening to clicks.
I think we need to decide first if we are gonna keep these buttons or not